exts/cesium.omniverse/cesium/omniverse/ui/credits_parser.py

```diff
@@ -84,4 +93,61 @@ def __init__(self, asset_credits: List[Tuple[str, bool]], should_show_on_screen:
             if perform_fallback:
                 self._logger.warning("Performing credits fallback.")
                 for credit, _ in asset_credits:
-                    ui.Label(credit, height=0, word_wrap=True)
+                    results.append(ParsedCredit(text=credit))
+
+        return results
+
+    def _build_ui_elements(self, parsed_credits: List[ParsedCredit], label_alignment: ui.Alignment):
+        for parsed_credit in parsed_credits:
+            if parsed_credit.image_uri:
+                if parsed_credit.link:
+                    CesiumImageButton(
+                        src=parsed_credit.image_uri,
+                        padding=4,
+                        clicked_fn=lambda: webbrowser.open_new_tab(parsed_credit.link),
```

Comment:
Since we're already breaking this stuff out. Lets break this out into a separate function that can be passed in rather than a lambda.

Reply:
The lambda is nice because it handles binding the parameter. I implemented it with `partial`. Is this sort of what you had in mind?

```python
@staticmethod
def _button_clicked(link: str):
    webbrowser.open_new_tab(link)

...

ui.Button(
    parsed_credit.text,
    clicked_fn=partial(self._button_clicked, parsed_credit.link),
    height=0,
    width=0,
)
```